src/Standard.Licensing/LicenseAttributes.cs

```diff
@@ -129,53 +133,57 @@ public virtual bool Contains(string key)
         /// <returns>true if the collection contains all specified elements; otherwise false.</returns>
         public virtual bool ContainsAll(string[] keys)
         {
-            return xmlData.Elements(childName).All(e => e.Attribute("name") != null && keys.Contains(e.Attribute("name").Value));
+            return XmlData.Elements(ChildName).All(e => e.Attribute("name") != null && keys.Contains(e.Attribute("name")?.Value));
         }
 
         protected virtual void SetTag(string name, string value)
         {
-            var element = xmlData.Element(name);
+            XElement element = XmlData.Element(name);
 
             if (element == null)
             {
                 element = new XElement(name);
-                xmlData.Add(element);
+                XmlData.Add(element);
             }
 
             if (value != null)
+            {
                 element.Value = value;
+            }
         }
 
         protected virtual void SetChildTag(string name, string value)
         {
-            var element =
-                xmlData.Elements(childName)
-                    .FirstOrDefault(e => e.Attribute("name") != null && e.Attribute("name").Value == name);
+            XElement element =
+                XmlData.Elements(ChildName)
+                    .FirstOrDefault(e => e.Attribute("name") != null && e.Attribute("name")?.Value == name);
```

Comment:
change from var, but also notice again here that the addition of the `?` is not needed as the null check was just performed